### ISSUE TITLE
BUG With color of lines when opening from link

### ISSUE THREAD

**Issue by scooler** [wokalski/covid-19charts.com#11]
When you open a chart from a link lines > 5th are gray

![image](https://user-images.githubusercontent.com/836023/77748062-07c77d00-7020-11ea-9e66-dffe0d437ad4.png)

But when you add or remove a country & chart gets rerendered - the colors show up:
![image](https://user-images.githubusercontent.com/836023/77748197-42311a00-7020-11ea-9551-4b0078f865b8.png)

**Comment by wokalski** [comment]
Right, obviously. It happens because the [`UseQueryParam` hook](https://github.com/wokalski/covid-19charts.com/blob/master/src/Index.re#L6-L11) doesn't use the customised setter. The proper solution would be allowing passing external state to `UseQueryParam` and use a reducer to handle colors and location updates at the same time.
